### ISSUE TITLE
pytddmon returning continuous "?ERROR" message when Jupyter Kernel running from folder

### ISSUE THREAD

**Issue by OscarSouth** [objarni/pytddmon#31]
![image](https://user-images.githubusercontent.com/32574713/37716482-92c83fd8-2d16-11e8-95d0-ac29035dbb71.png)

    return __import__(module, globals(), locals(), ["*"])
ModuleNotFoundError: No module named 'ipynb_checkpoints'

I'm pretty sure the cause is the 'ipynb_checkpoints' hidden file that Jupyter generates.  It happens both with Jupyter Notebook and Atom+Hydrogen.  When deleting these files from the folder then tests run correctly, but not sure how to deal with it as the folder is generated automatically when I edit the files with a Jupyter Lab text editor or Atom with Hydrogen running.

**Comment by objarni** [comment]
Yeah, pytddmon has no way of ignoreing / excluding paths. It's on the [wishlist](https://github.com/objarni/pytddmon/issues/30) though.
